src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -10,94 +10,180 @@ This page quickly describes how [Issues](https://guides.github.com/features/issu
+
+#### Backlog
+
+**All issues that are considered _ready_** by the lead Product Manager and the lead dev. Issues in this column will be discussed with Core developers.
+
+_Who adds issues to this column?_ It is the lead Product Manager with the approval of lead developers.
+
+#### To do
+
+**All the issues that are considered _ready_ and validated by the developers**, optionally assigned to one.
+
+_Who adds issues to this column?_ It is the lead Product Manager with the approval of Core developers. 
+
+#### Blocked / Need specs
+
+**All the issues which have been started but lack information and need to be respecified** or that are blocked by some other task.
+
+_Who adds issues to this column?_ It is the issue owner, after the Product Manager or developer assigned to the issue has explained the blocking element. A comment must be added to explain what is blocking and what is expected.
+
+#### In progress
+
+**All the issues that are being actively worked on**, assigned to a developer who has claimed it.
+
+_Who adds issues to this column?_ It is the assigned developer, once he/she starts working on it.
+
+#### To be reviewed
+
+**All the issues where a Pull Request has been made and is waiting for code review**
+
+_Who adds issues to this column?_ It is the developer working on the issue.
+
+#### To be tested
+
+**All the issues with an approved Pull Request that is waiting for validation** by the QA team.
+
+_Who moves issues to this column?_ It is the second maintainer who approves the Pull Request (PRs require at least two approvals), he/she must also add the "waiting for QA" label on the Pull Request.
+
+#### To be merged
+
+**All the issues with a Pull Request that is ready to be merged**, once approved by the QA team.
+
+_Who moves issues to this column?_ It is the QA analyst or the core developer that approves the Pull Request, he/she must also add the "QA approved" label on the Pull Request.
+
+#### Done
+
+All the issues that are now part of the history, great work to all! 
 
 ## Pull Requests
 
 When a contributor submits a Pull Request, it goes through multiple stages.
 
 ### Is it eligible
 
-Maintainers must first validate that the Pull Request is eligible to review (the template is filled, the license headers are correct, the target branch is the right one ...).
+Maintainers must first validate that the Pull Request is eligible to review (the template is filled, an issue is linked to the Pull Request, the license headers are correct, the target branch is the right one ...).
 
-If there is an issue with the Pull Request and it is not eligible, maintainers kindly ask the contributor to fix it.
+If the Pull Request is not eligible, maintainers kindly ask the contributor to fix it and apply the label "Waiting for author" on the Pull Request.
+If the issue associated to the Pull Request is in a project, then the issue should be moved in the "in progress" column.
```

Comment:
What do you mean by 'in a project'?

Reply:
In a github project, kanban if you prefer ?

Reply:
Maybe "Github project"